source/include/jobs.h

```diff
@@ -32,6 +32,7 @@
 #ifndef JOBS_H_
 #define JOBS_H_
 
+#include <stdbool.h>
```

Comment:
I personally disagree with including this.
Bools are going to likely be implemented as 8 bit values anyways.
Assuming that our APIs all currently return `size_t` variables. It feels unnecessary to include this new library just to have a single function be able to return a `bool` instead of a `size_t` like everything else

Returning a `size_t` allows for changes in the future with returning different error codes and the like
Using a bool now forces only one `pass` or `fail` status.

Reply:
> Using a bool now forces only one pass or fail status.

That is the point. For functions like `Jobs_IsStartNextAccepted` and `Jobs_IsJobUpdateStatus` it's a yes/no answer. Your checking for a topic match. It either matches or it does. Including stdbool seems like a much cleaner way to go that returning an enumeration which is essentially being used as a boolean. A user could treat the enum return incorrectly much more easily than they could a simple boolean.

> Returning a size_t allows for changes in the future with returning different error codes and the like

I can see absolutely no way in which a `is` prefixed function - a common convention for a boolean response - would need to change to return further response codes.